examples/http/azure_vm_list.js

```diff
@@ -0,0 +1,632 @@
+/**
+ * Domotz Custom Driver
+ * Name: Azure VMs List
+ * Description: Monitor Azure Compute Virtual Machines: this script retrieves information about Virtual Machine.
+ *
+ * Communication protocol is HTTPS
+ *
+ * Output:
+ * Extracts the following information from the data array:
+ *      - Name
+ *      - Resource Group
+ *      - OS
+ *      - Location
+ *      - Size
+ *      - Image Publisher
+ *      - Image
+ *      - Image Version
+ *      - Image SKU
+ *      - Data Disks
+ *      - Provisioning State
+ *      - Hibernation Enabled
+ *      - Managed Disk
+ *      - Computer Name
+ *      - Network Interfaces
+ *      - Zone
+ *      - Power State
+ *      - Network Out Total
+ *      - Network In Total
+ *      - OS Disk Write
+ *      - OS Disk Latency
+ *      - Percentage CPU
+ *      - OS Disk Read
+ *      - Temp Disk Latency
+ *      - Temp Disk Read
+ *      - Temp Disk Write
+ *      - Available Memory
+ *      - Disk Read
+ *      - Disk Write
+ *      - Data Disk Read
+ *      - Data Disk Write
+ *      - Data Disk Latency
+ *      - CPU Credits Remaining
+ *      - CPU Credits Consumed
+ *
+ **/
+
+// Parameters for Azure authentication
```

Comment:
There is a big inconsistency and mix of snake_case and camelCase for the parameter, stick to one